### PR TITLE
Fix now deployment alias and scaling issues

### PR DESCRIPTION
**Pull request by marcoow**
I scaled the apps once which should be permanently effective now. Also this makes sure the apps are deployed under the team account which is on a paid plan and supports scaling.

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -6,7 +6,6 @@
 /tmp
 /fastboot-dist
 /fastboot-tests
-/docs
```
Comment (marcoow) [inline .npmignore]: We actually need these files so we can run a production build on now which automatically builds the docs.

File: .travis.yml [hunk 2]
```diff
@@ -41,9 +41,9 @@ deploy:
     repo: simplabs/ember-simple-auth
 
 after_deploy:
-  - npm i -g now-pipeline
-  - now-pipeline-prune && now-pipeline --alias ember-simple-auth.now.sh
-  - cd server && now-pipeline-prune && now-pipeline --alias ember-simple-auth-server.now.sh
+  - npm i -g now
+  - now deploy --token=$NOW_TOKEN --team=e-simple-auth && now alias --token=$NOW_TOKEN --team=e-simple-auth
+  - cd server && now deploy --token=$NOW_TOKEN --team=e-simple-auth && now alias --token=$NOW_TOKEN --team=e-simple-auth
```
Comment (marcoow) [inline .travis.yml]: This is a bit repetitive but it works so I guess 🤷‍♂️

File: package.json [hunk 3]
```diff
@@ -24,6 +24,10 @@
     "test:fastboot": "ember fastboot:test",
     "test:node": "mocha node-tests --recursive"
   },
+  "now": {
+    "name": "ember-simple-auth",
+    "alias": "ember-simple-auth"
```
Comment (stevenwu) [inline package.json]: Ah I didn't know about these options. Should be `"alias": "ember-simple-auth.now.sh"`

**Comment by marcoow** [comment]
This does actually work though - I think `now.sh` is optional

Marco Otte-Witte
Founder/Director
marco.otte-witte@simplabs.com
http://simplabs.com

simplabs GmbH
Claude-Lorrain-Str. 7
81544 München
Geschäftsführer: Marco Otte-Witte
Handelsgericht München HRB 217887

**Pull request merged**